Components/Cards/Cards.css

```diff
@@ -0,0 +1,145 @@
+.card {
+  position: relative;
+  color: #ff3453;
+  background-color: #ffffff;
+  display: flex;
+  height: auto;
+  width: 25rem;
+  color: black;
+  border-radius: 1rem;
+  flex-direction: column;
+  box-shadow: 0 2px 1px -1px rgb(0 0 0 / 20%), 0 1px 1px 0 rgb(0 0 0 / 14%),
+    0 1px 3px 0 rgb(0 0 0 / 12%);
+}
+
+.card_horizontal {
+  position: relative;
+  color: inherit;
+  background-color: #ffffff;
```

Comment:
Suggestion : use color variable instead of hard code

Reply:
Sure